src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newObj = {};
+
+  for (const key in robot) {
+    if (newObj.hasOwnProperty(robot[key])) {
+      return null;
+    }
+
+    newObj[robot[key]] = key;
```

Comment:
let's create variable for `robot[key]`
you use this value twice in the code

Reply:
done